public/js/project.js

```diff
@@ -452,7 +452,7 @@ $('#delete-project').on('show.bs.modal', function(evt) {
 
   $('#delete-button').on('click', function() {
     $.ajax({
-      url: '/project/'+id,
+      url: '/project/'+ id,
```

Comment:
With one more space before `+` it would look better

Reply:
Fixed.